ckanext/rating/templates/rating/snippets/rating.html

```diff
@@ -1,27 +1,35 @@
 {#
-Renders a complete block of rating snippets with an option to rate the dataset
+Renders a complete block of rating snippets
 
 package  - The package for which the rating is displayed
 
     {% snippet "rating/snippets/rating.html", package=pkg %}
 #}
 
+{% resource "rating_css/rating.css" %}
 {% if h.show_rating_in_type(package.type) %}
-  <div class="rating">
-    {% block general_rating %}
-      <h2 class="heading">{{ _('Rating') }}</h2>
-      <div class="rating-container">
-        {% snippet "rating/snippets/stars_inactive.html", package=package %}
-      </div>
-    {% endblock %}
-    {% block user_rating %}
-      <h2 class="heading">{{ _('Your rating') }}</h2>
-      <div class="rating-container">
-        {%- snippet "rating/snippets/stars.html", package=package -%}<br>
-        <span class="rating-details">
-              {%- snippet "rating/snippets/rating_description.html", rating=h.get_user_rating(package.id) -%}
-            </span>
-      </div>
-    {% endblock %}
-  </div>
-{% endif %}
\ No newline at end of file
+    <div class="rating">
+      {% block general_rating %}
+        <h2 class="heading">{{ _('Rating') }}</h2>
+        <div class="rating-container">
+          {% snippet "rating/snippets/stars_inactive.html", package=package %}
+        </div>
+      {% endblock %}
+        {% block user_rating %}
+          {% if h.check_access('check_access_user')  %}
+            <h2 class="heading">{{ _('Your rating') }}</h2>
+            <div class="rating-container">
+              {%- snippet "rating/snippets/stars.html", package=package -%}
+              {% block user_rating_br %}<br>{% endblock %}
+              <span class="rating-details">
+                    {%- snippet "rating/snippets/rating_description.html", rating=h.get_user_rating(package.id) -%}
+                  </span>
+            </div>
+          {% else %}
+            <div class="login-rating-details">
+              <a href="{{h.url_for('login')}}">Login</a> to leave an App Rating
```

Comment:
I agree to leave "Login to leave a rating". What about localization - all file or only "Login to leave a rating" ?